### DISCUSSION TITLE
How to create a info box similar to mkdocs' frontpage?

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#3745] [Q&A] [answered]
Would like to make a info-box thing, similar to what mkdocs has on their frontpage.

The thing is, that they seem to use some classes that are part of the bootstrap CSS from the default theme?

Also, this clearly doesn't cover the fact of possible theme toggles like what is the case with Material here.

So, what should I consider when making such stuff?
I currently have a custom frontpage using theme extension and a `home.html` file that I use as template for the index page.

Current home.html:
https://paste.gg/p/anonymous/2d704d7217bd4376bde002a053ed5896

What I want to note is the usage of `{{ page.content }}` as I use the main content of the index.md file to fill, so I would preferably keep the info stuff in that file.

**Comment by squidfunk** [comment, marked as answer]
This is what card grids are for (see #3018). When this feature is released, you will be able to arrange all sorts of contents into grids, directly from Markdown of course with minimal additional syntax required.

If you want to add this yourself, [grid layout](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) is worth learning.

**Reply by Andre601** [reply]
This seems nice. Tho, I assume this will be a Insider feature until the goal - whichever it may be - is reached?

\*sigh* nothing comes easy these days... Will look into that grid thing a bit... Thanks for the info.

**Reply by squidfunk** [reply]
Yes, card grids will be an Insiders feature at first. It'll be likely on one of the higher goals, as the lower ones are already complete. Alternatively, you could use Bootstrap via additional CSS and use their classes to achieve something similar.
